projects/go-style-guide/src/app/features/ui-kit/components/toast-docs/toast-docs.component.html

```diff
@@ -46,6 +53,16 @@ <h4 class="go-heading-4">header</h4>
         </p>
       </div>
 
+      <div class="go-column go-column--100">
+        <h4 class="go-heading-4">icon</h4>
+        <p class="go-body-copy go-body-copy--no-margin">
+          Changes the icon that is displayed within the toast. This is set automatically for each type, but if a value
+          is passed for this binding it will override all of those type icons. See the
```

Comment:
So I see this is for the docs so I'm not sure it really matters, would it be a good idea to store text like this in a variable somewhere and use {{ interpolation }}?

Reply:
In this case, because it's just documentation and the value will never change I think it's fine to just have plain text here. Using interpolation on text that doesn't change is an extra thing that Angular will need to keep track of that it doesn't need to. Good thinking though!